target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4029-mr33.dts

```diff
@@ -60,7 +60,7 @@
 			status = "okay";
 			pinctrl-0 = <&mdio_pins>;
 			pinctrl-names = "default";
-			phy-reset-gpio = <&tlmm 47 0>;
+			phy-reset-gpio = <&tlmm 47 GPIO_ACTIVE_HIGH>;
```

Comment:
Ram's ipq-mdio patch that introduced phy-reset-gpio was never applied :(. So for the time being this could be removed.